src/elastio-snap.c

```diff
@@ -414,31 +414,7 @@ static void elastio_snap_bio_endio(struct bio *bio, int err){
 #define UMOUNT_NOFOLLOW 0
 #endif
 
-#if !defined(HAVE_BDEV_STACK_LIMITS) && !defined(HAVE_BLK_SET_DEFAULT_LIMITS)
-//#if LINUX_VERSION_CODE < KERNEL_VERSION(2,6,31)
```

Comment:
As we agreed in the offline discussion, we are stoppping to support 2nd kernels since we are started to support 6th kernels. So we are still keeping to support 4 major Linux kernel versions: 3 - 6.
The minimal Linux kernel version we do support in this driver is `3.10.0` of CentOS 7.

@skypodolsky you've removed compat `blk_set_default_limits` and its implementation for the kernels < `2.6.31`. Thus, let's update the minimal supported Linux kernel version in 2 places accordingly:
- https://github.com/elastio/elastio-snap/blob/master/INSTALL.md#dependencies
- https://github.com/elastio/elastio-snap/blob/7fe09e0082f13be0afe9789bd033e39151590ae8/src/elastio-snap.c#L12

Reply:
Done